app/assets/stylesheets/layout/_top_menu.sass

```diff
@@ -166,103 +166,119 @@ $search-input-width-expanded: 300px
-    z-index: 10
+    right: 2px
+    @include varprop(font-size, header-item-font-size)
     @include varprop(color, header-item-font-color)
+    &:hover
+      text-decoration: none
 
-.top-menu-search--button.-input-focused
-  color: $header-search-field-font-color
-
-input[type="text"].top-menu-search--input
-  margin: 0
-  padding: 0 20px 0 10px
-  display: inline-block
-  border-width: 1px
-  border-style: solid
-  @include varprop(border-color, header-item-font-color)
-  border-radius: 4px
-  height: 31px
-  width: $search-input-width
-  outline: 0
-  font-size: 0.9rem
-  background: transparent
-  @include varprop(color, header-item-font-color)
-  -webkit-transition: width 0.2s ease-in-out
-  transition: width 0.2s ease-in-out
-  &::-ms-clear
-    margin-right: 5px
-    width: 20px
-  &:focus
-    background: white
-    border-width: 1px
-    border-style: solid
-    @include varprop(border-color, header-item-font-color)
+  .top-menu-search--button.-input-focused
     color: $header-search-field-font-color
-    &:hover
-      background: white
-  &:hover
-    background: unset
-    border-width: 1px
-    border-style: solid
-    @include varprop(border-color, header-item-font-color)
 
-input.top-menu-search--input.-expanded
-  width: $search-input-width-expanded
-  background: white
-  border-width: 1px
-  border-style: solid
-  @include varprop(border-color, header-item-font-color)
-  color: $header-search-field-font-color
-  &:hover
-    background: white
+  .top-menu-search--loading
+    top: $header-height
+
+  .ng-select#q
+    width: $search-input-width
+    font-size: 0.9rem
+    -webkit-transition: width 0.2s ease-in-out
+    transition: width 0.2s ease-in-out
+    &::-ms-clear
+      margin-right: 5px
+      width: 20px
+
+    .ng-select-container
+      background: transparent
+      border-width: 1px
+      border-style: solid
+      @include varprop(border-color, header-item-font-color)
+      max-height: 32px
+      min-height: unset
+      &:hover
+        @include varprop(border-color, header-item-font-hover-color)
+
+    .ng-arrow-wrapper
+      opacity: 0
+
+    .ng-clear-wrapper
+      @include varprop(color, header-item-font-color)
+      top: 1px
 
-.search-autocomplete--results
-  width: 300px
+    .ng-input
+      top: 0
+      input
+        @include varprop(color, header-item-font-color)
+        height: 30px
+        cursor: text
+
+    .ng-placeholder
+      @include varprop(color, header-item-font-color)
+
+    &.-expanded
+      width: $search-input-width-expanded
+      background: white
+      border-radius: 4px
+      color: $header-search-field-font-color
+
+      .ng-select-container
+        border-radius: 4px !important
+
+      .ng-select-bottom
+        top: calc(100% + 12px)
+
+      .ng-placeholder,
+      .ng-clear-wrapper,
+      input
+        @include varprop(color, header-search-field-font-color)
+
+      .scroll-host
+        @include styled-scroll-bar-vertical
+        max-height: 50vh
```

Comment:
could we please try to use more height? I haven't tested it but maybe 80vh could be a good number.